src/features/loginGuard/LoginGuard.tsx

```diff
@@ -38,8 +34,13 @@ const LoginGuard: FC<{ children: ReactNode }> = ({ children }) => {
             Switch to {chains[0].name}
             {isLoading && pendingChainId === chains[0]?.id && ' (switching)'}
           </Button>
-          <div>{error && error.message}</div>
-        </Box>
+          {error && (
+            <div className="ml-1 mt-1.5 flex items-center text-red-500">
+              <AlertCircle size="14" />
+              <span className="ml-1 text-sm">{error.message}</span>
+            </div>
```

Comment:
For example when user rejects changing chain:

<img width="675" alt="Capture d’écran 2023-12-04 à 16 45 30" src="https://github.com/iExecBlockchainComputing/web3mail-usecase-demo/assets/4280765/275dbe48-2d4f-40ef-8caa-298f91a1eaee">